challenges/order_summary_component/index.html

```diff
@@ -25,39 +32,101 @@
       sizes="16x16"
       href="../images/favicon-16x16.png"
     />
-    <link rel="manifest" href="../site.webmanifest" />
-    <link rel="stylesheet" href="style.css" />
+
+    <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
+    <style>
+      .attribution {
+        font-size: 11px;
+        text-align: center;
+      }
+      .attribution a {
+        color: #3e52a3;
+      }
+      .attribution pale-blue {
+        color: #e0e8ff;
+      }
+      .attribution bright-blue {
+        color: #3829e0;
+      }
+      .attribution very-pale-blue {
+        color: #f5f7ff;
+      }
+      .attribution desatured-blue {
+        color: #7280a7;
+      }
+      .attribution dark-blue {
+        color: #1f2f56;
+      }
+    </style>
   </head>
-  <body class="m-0">
+  <body
+    class="font-red-hat-display bg-bright-blue bg-hero-pattern h-screen bg-contain bg-no-repeat text-base"
+  >
     <div
-      class="flex h-[100vh] w-full flex-col place-content-center text-center"
+      class="lg:w-400 md:m-w-400 sm:w-350p container mr-auto ml-auto mt-24 grid max-w-md place-items-center overflow-hidden rounded-lg bg-white text-center"
```

Comment:
No need for `sm:` prefix as per [mobile-first design principles](https://tailwindcss.com/docs/responsive-design#mobile-first).